zlib-ng.mk

```diff
@@ -0,0 +1,55 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += zlib-ng
+ZLIB-NG_VERSION  := 2.0.2
+DEB_ZLIB-NG_V    ?= $(ZLIB-NG_VERSION)
+
+zlib-ng-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://github.com/zlib-ng/zlib-ng/archive/refs/tags/$(ZLIB-NG_VERSION).tar.gz
+	$(call EXTRACT_TAR,$(ZLIB-NG_VERSION).tar.gz,zlib-ng-$(ZLIB-NG_VERSION),zlib-ng)
+ifneq ($(wildcard $(BUILD_WORK)/zlib-ng/.build_complete),)
+zlib-ng:
+	@echo "Using previously built zlib-ng."
+else
+zlib-ng: zlib-ng-setup
+	cd $(BUILD_WORK)/zlib-ng && cmake . \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_SYSTEM_NAME=Darwin \
+                -DCMAKE_CROSSCOMPILING=true \
+                -DCMAKE_INSTALL_NAME_TOOL=$(I_N_T) \
+                -DCMAKE_INSTALL_PREFIX=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) \
+                -DCMAKE_INSTALL_NAME_DIR=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib \
+                -DCMAKE_INSTALL_RPATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) \
+                -DCMAKE_OSX_SYSROOT="$(TARGET_SYSROOT)" \
+                -DCMAKE_C_FLAGS="$(CFLAGS)" \
+                -DCMAKE_CXX_FLAGS="$(CXXFLAGS)" \
+                -DCMAKE_FIND_ROOT_PATH=$(BUILD_BASE) \
+				                -DCMAKE_OSX_ARCHITECTURES="$(MEMO_ARCH)" \
+								ZLIB_COMPAT=TRUE \
```

Comment:
Remove this, it will install zlib-nq as `libz.dylib` instead of `libz-ng.dylib` which is exactly what we don't want.

Reply:
> Remove this, it will install zlib-ng as `libz.dylib` instead of `libz-ng.dylib` which is exactly what we don't want.


That is incorrect, 
Doing `$ls` in the lib folder yields the following results:
```
libz-ng.2.0.2.dylib  libz-ng.2.dylib  libz-ng.a  libz-ng.dylib  pkgconfig
```

Reply:
That's cause you didn't do `-DZLIB_COMPAT`, it should still be removed.

Reply:
chillax